app/assets/javascripts/dashboard/explorative_annotations_view.js

```diff
@@ -357,7 +357,7 @@ class ExplorativeAnnotationsView extends React.PureComponent<Props, State> {
         <Column
           title="Stats"
           render={(__, tracing: APIAnnotationType) =>
-            tracing.stats && tracing.content.typ === "skeleton" ? (
+            tracing.stats.treeCount && tracing.content.typ === "skeleton" ? (
```

Comment:
Actually, I wanted to change this to `treeCount != null`, since this won't display the statistics when the tree count is zero. However, this does not typecheck (I opened an issue in the flow repo) **and** the stats object is empty anway, when there aren't any trees (and thus no nodes or edges anyway).

However, I think, the cleanest solution would be that the server always delivers a `stats` object for skeleton tracings and `null` for volume tracings. Even for empty tracings, `stats` should contain attributes (which are `0` then). @fm3 Would this be an easy change? It's not high-pri und doesn't need to block this PR. But if it's easy (and if it makes sense to you), you can change that at some point?

Reply:
not super easy since the stats object is just any json object to the wK backend. it is written only upon update actions via the datastore. but of course we can somehow add a default json object there.

Reply:
ok, I see. then, let's leave it as is :+1: